### ISSUE TITLE
[bug] Email scanner crashes every day

### ISSUE THREAD

**Issue by aleterracciano** [YetiForceCompany/YetiForceCRM#17326]
Good morning,
Even though it worked for a long time, suddenly the email scanner, which I use to transform emails into tickets, keeps crashing. I can't understand what caused it. Error icons have appeared indicating a problem with the mailboxes but the emails are received and sent correctly. Can you help me?

Below are the system configuration screens and email scanners.

Thank you

![image](https://github.com/YetiForceCompany/YetiForceCRM/assets/53528784/5530a0d5-c052-45f6-ab4f-e5f4c1e30785)

![image](https://github.com/YetiForceCompany/YetiForceCRM/assets/53528784/c4d076e9-1be1-4369-a211-6db4b14f2c16)

I attach system configuration

![ConfReport (11)](https://github.com/YetiForceCompany/YetiForceCRM/assets/53528784/f7d7b8e3-2b64-4ba6-89c1-92d2050d407d)

![image](https://github.com/YetiForceCompany/YetiForceCRM/assets/53528784/ed541317-c0b8-4388-9759-ab4fd9482c57)

after a while it becomes like this!

If I start cron.php manually via the address www.miosito.it/cron.php this appears:

---------------  2023-11-06 11:27:20 | Start CRON  ----------
2023-11-06 11:27:20 | LBL_WORKFLOW - Not ready to run as the time to run again is not completed
2023-11-06 11:27:20 | LBL_SCHEDULED_IMPORT - Not ready to run as the time to run again is not completed
2023-11-06 11:27:20 | LBL_MAIL_SCANNER_ACTION - Not ready to run as the time to run again is not completed
2023-11-06 11:27:20 | LBL_MAIL_SCANNER_VERIFICATION - Start task
2023-11-06 11:27:20 | LBL_MAIL_SCANNER_VERIFICATION - End task (0.02 s)
2023-11-06 11:27:20 | LBL_MAIL_SCANNER_BIND - Not ready to run as the time to run again is not completed
2023-11-06 11:27:20 | LBL_MAILER - Not ready to run as the time to run again is not completed
2023-11-06 11:27:20 | LBL_ACTIVITY_STATE - Not ready to run as the time to run again is not completed
2023-11-06 11:27:20 | LBL_SEND_REMINDER - Not ready to run as the time to run again is not completed
2023-11-06 11:27:20 | LBL_SEND_NOTIFICATIONS - Start task
2023-11-06 11:27:20 | LBL_SEND_NOTIFICATIONS - End task (0.01 s)
2023-11-06 11:27:20 | LBL_MULTI_REFERENCE_VALUE - Not ready to run as the time to run again is not completed
2023-11-06 11:27:20 | LBL_CRMACTIVITY_DAYS - Not ready to run as the time to run again is not completed
2023-11-06 11:27:20 | LBL_RECORD_LABEL_UPDATER - Not ready to run as the time to run again is not completed
2023-11-06 11:27:20 | LBL_PRIVILEGES_UPDATER - Not ready to run as the time to run again is not completed
2023-11-06 11:27:20 | LBL_ADDRESS_BOOK - Not ready to run as the time to run again is not completed
2023-11-06 11:27:20 | LBL_MARK_RECORDS_AS_REVIEWED - Not ready to run as the time to run again is not completed
2023-11-06 11:27:20 | LBL_CACHE - Not ready to run as the time to run again is not completed
2023-11-06 11:27:20 | LBL_NEVER_ENDING_RECURRING_EVENTS - Not ready to run as the time to run again is not completed
2023-11-06 11:27:20 | LBL_CLEAR_FILE_UPLOAD_TEMP - Not ready to run as the time to run again is not completed
2023-11-06 11:27:20 | LBL_BROWSING_HISTORY - Not ready to run as the time to run again is not completed
2023-11-06 11:27:20 | LBK_SYSTEM_WARNINGS - Not ready to run as the time to run again is not completed
2023-11-06 11:27:20 | LBL_BATCH_PROCESSES - Not ready to run as the time to run again is not completed
2023-11-06 11:27:20 | LBL_BATCH_METHODS - Start task
2023-11-06 11:27:20 | LBL_BATCH_METHODS - End task (0 s)
2023-11-06 11:27:20 | LBL_SESSION_CLEANER - Start task
2023-11-06 11:27:20 | LBL_SESSION_CLEANER - End task (0.01 s)
2023-11-06 11:27:20 | LBL_CURRENCY_UPDATE - Not ready to run as the time to run again is not completed
2023-11-06 11:27:20 | LBL_MAIL_RBL - Not ready to run as the time to run again is not completed
===============  2023-11-06 11:27:20 (0.06) | End CRON  ==========

Ho analizzato la tabella vtiger_cron_task nel database e nella colonna lase_error del processo LBL_WORKFLOW ho trovato il seguente testo: 
ERR_ILLEGAL_FIELD_VALUE||email||Vendors||.

![image](https://github.com/YetiForceCompany/YetiForceCRM/assets/53528784/2bba0a9f-2432-41f7-83bb-fb8fe3c52c86)

Even if I start manually, it crashes.

I can't figure out how to unlock it.
Can you help me?

**Comment by aleterracciano** [comment]
Good morning, today I also encountered the following error. When I try to manually start the scan, the cron crashes.

![image](https://github.com/YetiForceCompany/YetiForceCRM/assets/53528784/b4805eb4-fb51-48b6-a0c0-04cf6f20a186)

**Comment by aleterracciano** [comment]
Good morning, I hope this info can help someone, I solved the email scanner problem in the following way. 
Although the mailboxes worked fine, receiving and sending emails, the scanner did not work well due to the smtp configuration.
I solved it by changing the Roundcube mail configuration.
Instead of port 587 I inserted port 465 but changing the server address in SMTP from "mail.vianova.it" to "ssl://mail.vianova.it" In this way the server allowed the correct connection and the errors in the scanner have disappeared. 
By changing only the port, the system did not accept the changes, working only with port 587, only after changing the address did the system start to work correctly. I will wait a couple of days to close this ticket.

![image](https://github.com/YetiForceCompany/YetiForceCRM/assets/53528784/f6818568-10e2-449f-b0b6-ec948812056d)
